tests/nextflow-tests/run_tests_human.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SOURCE_DIR="$(dirname $(dirname $SCRIPT_DIR))/nextflow"
+
+cwd=${PWD}
+cd ${SCRIPT_DIR}
+mkdir project project/accessions project/public ftp
+
+# run accession and variant load
+# note public_dir needs to be an absolute path, unlike others in config
+printf "\e[32m===== ACCESSION PIPELINE =====\e[0m\n"
+nextflow run ${SOURCE_DIR}/accession.nf -params-file test_ingestion_config_human.yaml \
+   --accessions_dir ${SCRIPT_DIR}/project/accessions \
+	 --public_dir ${SCRIPT_DIR}/project/public
+printf "\e[32m==== VARIANT LOAD PIPELINE ====\e[0m\n"
+nextflow run ${SOURCE_DIR}/variant_load.nf -params-file test_ingestion_config_human.yaml \
+   --project_dir ${SCRIPT_DIR}/project
+
+# check for public files and logs
+printf "\e[32m====== Files made public ======\e[0m\n"
+ls project/public
```

Comment:
Is this testing that the directory is empty ?

Reply:
This isn't really testing anything except that the nextflow runs without error, it's just printing out what it does so the developer can sanity check it.  We should definitely make it so that the test does the sanity check rather than the human.  I can make a ticket.

Reply:
Ok, it makes sense. And is the same as the other script. 
We should improve this though. Please create the ticket for it alongside [deduplication](https://github.com/EBIvariation/eva-submission/pull/33/files#r625919323) of the tests.